src/powershell/Public/ConvertTo-FinOpsSchema.ps1

```diff
@@ -0,0 +1,214 @@
+    Specifies the path of the output CSV file.
+
+    .PARAMETER RunTest
+    Specifies whether to run a test to estimate the processing time.
+
+    .PARAMETER rows
+    Specifies the number of rows to use for the test.
+
+    .PARAMETER columns
+    Specifies the number of columns to use for the test.
+
+    .PARAMETER ExportAllColumns
+    Specifies whether to export all columns or only the columns specified in the column mapping.
+
+    .PARAMETER Force
+    Specifies whether to force the operation to continue without user confirmation after the test is complete.
+
+    .EXAMPLE
+    .\ConvertTo-FinOpsSchema.ps1 -ActualCost .\Detail_Account_196312_202309_en.csv -outputFilePath .\Detail_Account_196312_202309_en_filtered.csv -ExportAllColumns $false -Force
+
+    # This will convert the input file to the output file using the default column mapping and export only the columns specified in the column mapping. The operation will not run a test and will not prompt the user to continue.
+
+    .EXAMPLE
+    .\ConvertTo-FinOpsSchema.ps1 -ActualCost .\Detail_Account_196312_202309_en.csv -outputFilePath .\Detail_Account_196312_202309_en_filtered.csv -ExportAllColumns $true -Force
+
+    # This will convert the input file to the output file using the default column mapping and export all columns. The operation will not run a test and will not prompt the user to continue. 
+
+    .EXAMPLE
+    .\ConvertTo-FinOpsSchema.ps1 -ActualCost .\Detail_Account_196312_202309_en.csv -outputFilePath .\Detail_Account_196312_202309_en_filtered.csv -ExportAllColumns $true -RunTest $true -rows 1000 -columns 10
+    # This will run a test to estimate the processing time and then prompt the user to continue. If the user enters Y or y, the operation will continue. If the user enters N or n, the operation will be aborted. 
+
+    .EXAMPLE
+    .\ConvertTo-FinOpsSchema.ps1 -ActualCost .\Detail_Account_196312_202309_en.csv -outputFilePath .\Detail_Account_196312_202309_en_filtered.csv -ExportAllColumns $false -RunTest $true -rows 1000 -columns 10 -Force
+    # This will run a test to estimate the processing time and then continue without prompting the user. It will export only the columns specified in the column mapping.
+#>
+
+<# Parameter Block #>
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $ActualCost,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $outputFilePath,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $RunTest = $false,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateRange(0, 100000)]
+    [int]
+    $rows = 1000,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateRange(0, 20)]
+    [int]
+    $columns = 10,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $ExportAllColumns = $true,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $Force
+)
+
+# Define the input and output file paths
+$ActualCost=".\Detail_Account_196312_202309_en.csv"
+$outputFilePath=".\Detail_Account_196312_202309_en_filtered.csv"
```

Comment:
Are these for local testing? Maybe create a test script that sets them instead of adding them directly into the script.